src/ci/utils.py

```diff
@@ -142,111 +203,127 @@ def checkCoverage(output):
+        currentDir = os.path.join(currentDirPath(moduleName), 'build')
     else:
-        currentDir = currentBuildDir + str(moduleName) + "/build/bin/"
+        currentDir = os.path.join(currentDirPath(moduleName), 'build/bin')
+
     objects = os.scandir(currentDir)
     for entry in objects:
         if entry.is_file() and bool(re.match(reg, entry.name)):
             tests.append(entry.name)
     valgrindCommand = "valgrind --leak-check=full --show-leak-kinds=all -q --error-exitcode=1 " + currentDir
+
     for test in tests:
-        out = subprocess.run(valgrindCommand + test, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+        out = subprocess.run(os.path.join(valgrindCommand, test), stdout=subprocess.PIPE,
+                             stderr=subprocess.PIPE, shell=True)
         if out.returncode == 0:
-            printGreen('[' + test + ': PASSED]')
+            printGreen(f'[{test} : PASSED]')
         else:
             print(out.stdout)
             print(out.stderr)
-            printFail('[' + test + ': FAILED]')
+            printFail(f'[{test} : FAILED]')
             errorString = 'Error Running valgrind: ' + str(out.returncode)
             raise ValueError(errorString)
 
+
 def runCoverage(moduleName):
     """
     Executes code coverage under 'moduleName' lib unit tests.
 
     :param moduleName: Lib to be analyzed using gcov and lcov tools.
     """
-    currentDir = currentBuildDir + str(moduleName)
-    reportFolder = currentDir + '/coverage_report'
-
+    currentDir = currentDirPath(moduleName)
+    reportFolder = os.path.join(currentDir, 'coverage_report')
+    includeDir = Path(currentDir)
     moduleCMakeFiles = ""
-    excludeTests = ""
+
     if moduleName == 'shared_modules/utils':
-        moduleCMakeFiles = currentDir + "/tests/*/CMakeFiles/*.dir"
+        moduleCMakeFiles = os.path.join(currentDir, '*/CMakeFiles/*.dir')
+        includeDir = includeDir.parent
     else:
-        moduleCMakeFiles = currentDir + "/build/tests/*/CMakeFiles/*.dir"
-        excludeTests = '--exclude "*/tests/*"'
+        moduleCMakeFiles = os.path.join(
+            currentDir, 'build/tests/*/CMakeFiles/*.dir')
 
-    printHeader("<"+moduleName+">"+headerDic['coverage']+"<"+moduleName+">")
+    printHeader(moduleName, 'coverage')
     folders = ''
     if not os.path.exists(reportFolder):
         os.mkdir(reportFolder)
 
     for dir in glob.glob(moduleCMakeFiles):
         folders += '--directory ' + dir + ' '
-    includeDir = currentDir.replace("ci/../","")
-    coverageCommand = 'lcov ' + folders + ' --capture --output-file ' + reportFolder + '/code_coverage.info -rc lcov_branch_coverage=0 --exclude="*/tests/*" --include "'+includeDir+'/*" -q'
+
+    coverageCommand = f'lcov {folders} --capture --output-file {reportFolder}/code_coverage.info -rc ' \
+                      f'lcov_branch_coverage=0 --exclude="*/tests/*" --include "{includeDir}/*" -q'
```

Comment:
PEP8 linter show "continuation line over-indented for visual indent" [E127](https://www.flake8rules.com/rules/E127.html)
```suggestion
    coverageCommand = f'lcov {folders} --capture --output-file {reportFolder}/code_coverage.info -rc ' \
                       f'lcov_branch_coverage=0 --exclude="*/tests/*" --include "{includeDir}/*" -q'
```

Reply:
Hi agail @antoniomanuelfr ,

There are multiples python linters (flake8, pylint), and only in one of them his change was suggested. But this change generate alerts in the other ones, so i will mark this as solved.

Sorry about this misunderstanding. 

Regards,
Nico